src/main/java/mServer/crawler/sender/MediathekArd.java

```diff
@@ -274,44 +290,17 @@ private void filmSuchen1(String strUrlFeed, String thema, boolean weiter) {
                         urlSendung = urlSendung.replace("&amp;", "&");
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Remove this useless assignment to local variable "urlSendung". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1854)